flow/flow.go

```diff
@@ -620,7 +712,7 @@ func SetReceiver(portId uint16) (OUT *Flow, err error) {
 	}
 	createdPorts[portId].wasRequested = true
 	createdPorts[portId].willReceive = true
-	rings := low.CreateRings(burstSize * sizeMultiplier, createdPorts[portId].InIndex)
+	rings := low.CreateRings(burstSize*sizeMultiplier, createdPorts[portId].InIndex)
```

Comment:
It is an important commit - so I propose to remove gofmt from here. You can make clearing commit from other branch and then rebase at this commit.